README.rst

```diff
@@ -101,11 +101,12 @@ You can install or upgrade ``python-telegram-bot`` via
 
 .. code:: shell
 
+    #Requires Python 3.10+.
     $ pip install python-telegram-bot --upgrade
 
 To install a pre-release, use the ``--pre`` `flag <https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-pre>`_ in addition.
 
-You can also install ``python-telegram-bot`` from source, though this is usually not necessary.
+You can also install ``python-telegram-bot`` from source, though this is usually not necessary.  Before building from source, check that Pything headers and a C compiler are installed.
```

Comment:
PTB is a pure-python package and doesn't have any C-bindings. What do you thnik Python headers & C compilers are required for?